pkg/didcomm/protocol/didexchange/states.go

```diff
@@ -492,3 +510,15 @@ func (ctx *context) handleInboundResponse(response *Response) (stateAction, erro
 func getEpochTime() int64 {
 	return time.Now().Unix()
 }
+func getPublicKey(didDoc *did.Doc, pubKeyType string) ([]did.PublicKey, error) {
```

Comment:
imo the function name should end with` s` ie, `getPublicKeys` as this is returning a slice.

Reply:
done